.github/workflows/docs-build-pr.yml

```
@@ -42,7 +42,7 @@ jobs:
     - name: Set up Python 3
       uses: actions/setup-python@v4
       with:
-        python-version: 3
+        python-version: "3.10"
```

What's the need for an older version? 3 was specifically intended to avoid pinning. The default Python action marches along at a reasonably slow pace, so we shouldn't need to worry about 3.12 becoming default before the ecosystem is ready, but by now it should be safe to build on 3.11.

---

Not sure why I did this -- rolling back

```suggestion
        python-version: 3
```